views/design.jade

```diff
@@ -18,6 +18,6 @@ block content
                     +inputField('glob.company', glob, 'Your Company Name:')
                     +inputField('glob.footer', glob, 'Portal Page Footer:')
             
-            +renderButtons('/design', '/templates', null)
+            +renderButtons('/design', '/templates', '/kongadaptor')
```

Comment:
`/kongadapter` please, to keep it consistent.

Reply:
It would actually not be correct even

Reply:
Ah! sorry i thought i modified all the occurrence.